src/components/NavBar/index.tsx

```diff
@@ -107,7 +108,9 @@ function NavBar({ classes }: NavBarProps) {
                     <Grid container justify="space-around" item>
                       {buttons}
                     </Grid>
-                    <Grid item>{menu}</Grid>
+                    <Grid container direction="column" item>
+                      <MenuItemMobile />
```

Comment:
`<MenuItem/>` took props for the menu items, but `<MenuItemMobile/>` doesn't? For consistency's sake we should follow a similar approach to `<MenuItem/>` and define a `menu` constant for mobile. Or, we can rename both `MenuItem` and `MenuItemMobile` to `MenuItem*s*...` so its more clear they are multiple items. :wink:

Another approach: We could use `useBreakpoint` and based off whether we are on mobile or not, we can replace the use of `MenuItem` with `MenuItemMobile` on line 40. This means we don't have to touch this part of the file at all.

Reply:
Let me know if you have any questions :+1:

Reply:
I have updated the codes like I mentioned in [here](https://github.com/WFP-VAM/prism-frontend/pull/129#discussion_r595683092). Will need your help to check it again after I push the commit 🙏